Process.c

```diff
@@ -1274,13 +1274,14 @@ void Process_updateFieldWidth(ProcessField key, size_t width) {
 }
 
 void Process_updateCPUFieldWidths(float percentage) {
-   if (percentage < 99.9) {
-      Process_updateFieldWidth(PERCENT_CPU, 4);
-      Process_updateFieldWidth(PERCENT_NORM_CPU, 4);
+   if (percentage < 999.9F) {
+      Process_updateFieldWidth(PERCENT_CPU, 5);
+      Process_updateFieldWidth(PERCENT_NORM_CPU, 5);
```

Comment:
No need to increase this as this is already captured by the minimum width enforced by the column title.

Reply:
If column title enforce minimum width of the respective column, why do we need to update field width using `Process_updateFieldWidth` function? Title width of `CPU% ` and `NCPU%` is already `5`, so can't we do something like this:

```c
if (percentage < 999.9F) {
   return;
}
```

This would make sure, we don't do number of characters calculation, if percentage is less than `999.9F`.

Reply:
Actually most values are below 99.9, thus having that case specially hadled by setting it to 4 takes care of most calls. For 999.9 you want that distinction, because there potentially are columns with floating point values with just 4 digits and those would need the handling of 4 vs. 5 digits.

Reply:
Sorry, I still didn't get it.

> thus having that case specially hadled by setting it to 4 takes care of most calls.

Why do we need to handle (/update) field width, at all? If we're sure minimum width (of `CPU% ` and `NCPU%` is `5`) which is greater than `4`, we can ignore `Process_updateFieldWidth()` function call, when value is less than `99.9F`. And do something like this:

```patch
 void Process_updateCPUFieldWidths(float percentage) {
-   if (percentage < 99.9) {
-      Process_updateFieldWidth(PERCENT_CPU, 4);
-      Process_updateFieldWidth(PERCENT_NORM_CPU, 4);
+   if (percentage < 999.9F) {
       return;
    }
```

Reply:
As there are some columns with minimum of 4 digits you can't do this simplification without forcing all columns to minimum of 5 characters.

Also, including those two calls doesn't hurt, as the `Process_updateFieldWidth` does no expensive processing to update the minimum width value.